livedata/src/main/java/io/getstream/chat/android/livedata/service/sync/BackgroundSyncConfigStore.kt

```diff
@@ -0,0 +1,59 @@
+package io.getstream.chat.android.livedata.service.sync
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.security.crypto.EncryptedSharedPreferences
+import androidx.security.crypto.MasterKey
+
+interface BackgroundSyncConfigStore {
```

Comment:
Do we really need an interface here?

Reply:
Maybe later if it's injected into some usecase to allow mocking. Now I'll remove it.